modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/constants/pageSpeed.js

```diff
@@ -12,6 +12,6 @@
  * details.
  */
 
-export const API_KEY_ERROR_CODE = 400;
-export const API_KEY_INVALID_STATUS = 'INVALID_ARGUMENT';
-export const SERVER_ERROR_CODE = 500;
+export const GPS_API_KEY_ERROR_CODE = 400;
```

Comment:
I think the `GPS` prefix is a bit confusing 😛 if we want to prefix them, why not using `PAGE_SPEED`?